src/scanner/rule-to-links-mappings.ts

```diff
@@ -87,4 +87,8 @@ export const ruleToLinkConfiguration: DictionaryStringTo<HyperlinkDefinition[]>
     'aria-allowed-role': [BestPractice],
     'autocomplete-valid': [BestPractice],
     'css-orientation-lock': [BestPractice],
+    'aria-hidden-focus': [link.WCAG_4_1_2],
+    'form-field-multiple-labels': [BestPractice],
+    'label-content-name-mismatch': [BestPractice],
+    'landmark-complementary-is-top-level': [link.WCAG_1_3_1],
```

Comment:
>'landmark-complementary-is-top-level': [link.WCAG_1_3_1], [](start = 4, length = 57)

Can you mark this as `[link.WCAG_1_3_1, BestPractice]` off the bat, per #390?

Reply:
Done 😄